code/game/objects/effects/projectile/projectile_effects.dm

```diff
@@ -0,0 +1,54 @@
+/obj/effect/projectile
+	name = "pew"
+	icon = 'icons/obj/projectiles.dmi'
+	icon_state = "nothing"
+	layer = 4.5
+	anchored = TRUE
+	unacidable = TRUE
+	light_color = "#00ffff"
+	mouse_opacity = 0
+	appearance_flags = 0
+
+/obj/effect/projectile/singularity_pull()
+	return
+
+/obj/effect/projectile/singularity_act()
+	return
+
+/obj/effect/projectile/proc/scale_to(nx,ny,override=TRUE)
+	var/matrix/M
+	if(!override)
```

Comment:
Nit: flip the arms of this if to remove the not.

Reply:
What do you mean by this?